packages/metro/src/DeltaBundler/traverseDependencies.js

```diff
@@ -190,6 +190,16 @@ async function traverseDependenciesForSingleFile<T>(
   options.onDependencyAdded();
 }
 
+function absolutePathHasChanged(
+  dependencyA: Dependency,
+  dependencyB: Dependency,
+): Boolean {
+  if (dependencyA == null || dependencyB == null) {
```

Comment:
Just wondering, when would `dependencyB` be `null`?

Reply:
I decided to remove this function and just do the check inline for simplicity